pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -22,11 +22,15 @@
     <!-- Exclude shims from the closure verification -->
     <ExcludeFromClosure Include="mscorlib" />
```

Comment:
yeah I was thinking about that but there is a decent chance we may change the set of shims in UAP so I kept them separate for now. We can eliminate the duplication at some point as clean-up.